src/tests/system/tests/test_sss_override.py

```diff
@@ -218,14 +222,26 @@ def test_sss_overrides__export_then_import_user_and_group_override_data(client:
     client.sssd.restart()
     assert not client.sss_override.user("user1").get()
     assert not client.sss_override.group("group1").get()
+
+    assert (
+        len(
+            client.ldb.search(
+                f"/var/lib/sss/db/cache_{provider.domain},kdb",
+                f"cn={provider.domain},cn=sysdb",
+                filter="objectClass=userOverride",
+            ).items()
+        )
+        < 1
+    ), "Override is not not empty!"
```

Comment:
I personally have no problem with this but, considering the black-box testing concept, we should not be doing this. So I don't know what to tell you. Is there a clear directive about performing black-box testing only?

BTW, typo -> "not not empty"

Reply:
For right now, let's just leave, as long as the test is okay, it's easy to refactor later.